datalad/tests/test_annexrepo.py

```diff
@@ -41,10 +41,11 @@ def test_AnnexRepo_instance_from_clone(src, dst):
     # there's already a git-repo at that path and therefore can't clone to `dst`
     with swallow_logs() as cm:
         assert_raises(GitCommandError, AnnexRepo, dst, src)
-        if git.__version__ != "1.0.2":
+        if git.__version__ != "1.0.2" and git.__version__ != "2.0.5":
```

Comment:
so will it back to "normal" in 2.0.6 -- i.e. is it anyhow resolved in GitPython master?